release-centos7-llvm/scripts/build-tiflash-release.sh

```diff
@@ -74,13 +80,19 @@ cmake -S "${SRCPATH}" \
   -DENABLE_PCH=${ENABLE_PCH} \
   -GNinja
 
-cmake --build . --target tiflash --parallel ${NPROC}
-cmake --install . --component=tiflash-release --prefix="${INSTALL_DIR}"
-
-# unset LD_LIBRARY_PATH before test
-unset LD_LIBRARY_PATH
-readelf -d "${INSTALL_DIR}/tiflash"
-ldd "${INSTALL_DIR}/tiflash"
-
-# show version
-${INSTALL_DIR}/tiflash version
+if [ $CMAKE_BUILD_TYPE == "TSAN" ] || [ $CMAKE_BUILD_TYPE == "ASAN" ]; then
+  cmake --build . --target gtests_dbms --parallel ${NPROC}
```

Comment:
```suggestion
  cmake --build . --target gtests_dbms gtests_libcommon --parallel ${NPROC}
```
Could also build `gtests_libcommon`